http/file_server.ts

```diff
@@ -645,12 +656,18 @@ async function createServeDirResponse(
   }
 
   if (showDirListing) { // serve directory list
-    return serveDirIndex(fsPath, { showDotfiles, target });
+    return serveDirIndex(fsPath, { showDotfiles, target, quiet });
   }
 
   return createCommonResponse(Status.NotFound);
 }
 
+function logError(error: unknown) {
+  console.error(
+    red(error instanceof Error ? error.message : "[non-error thrown]"),
```

Comment:
Not directly related to this change, but `"[non-error thrown]"` message doesn't look useful. How about just printing `error` itself?

Reply:
Makes sense. It has changed! (https://github.com/denoland/deno_std/pull/3371/commits/d8137cb34cc33182d830e1c8cfc94700832c700c)